src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -77,7 +77,14 @@
 @end
 
 # Generate a simple argument
-@private initLineSimple(line)
+@private initLineSimple(line, idx)
+  @if line.doc
+    @if util.toInt(idx) > util.toInt(0)
```

Comment:
We need to keep logic out of the templates - try to move this logic into transformers.

Reply:
Sure, I tried. @geigerj suggested this could also be done as a post-processing step in artman.